src/views/AddItem.jsx

```diff
@@ -12,17 +12,67 @@ export function AddItem({ listToken }) {
 		addItemToList();
 	};
 
+	// Removes excess spaces or special characters, and makes all leters lowercase.
+	const normalizeText = (item) => {
+		return item
+			.trim() // removes space before and after item
+			.split(/ +/) // removes extra spaces between words
+			.join(' ')
+			.toLowerCase() // makes text all lowercase
+			.replace(/[^a-z0-9]/gi, ''); // remove punctuation and special characters
+	};
+
+	// Checks if user's input item already exists in the database.
+	const checkIfItemExists = (item) => {
```

Comment:
I think we can get rid of the parameter in this function, since we're not actually using `item` in the function.

Reply:
I agree with this as well. Might as well save yourself a little computing.